unittest/buildtool/git_test.py

```diff
@@ -110,6 +110,31 @@ def test_determine_tag_at_head(self):
         self.assertEquals([], messages)
         self.assertEquals(version, tag)
 
+  def test_same_repo(self):
+    variants = [
+        'http://github.com/user/spinnaker',
+        'http://github.com/user/spinnaker.git',
+        'https://github.com/user/spinnaker',
+        'https://github.com/user/spinnaker.git',
+        'git@github.com:user/spinnaker.git',
+        'git@github.com:user/spinnaker.git'
```

Comment:
Local path is missing for `True` - not supported?

Reply:
not applicable